tests/src/e2e/helpers.rs

```diff
@@ -134,3 +135,25 @@ pub fn get_epoch(test: &Test, ledger_address: &str) -> Result<Epoch> {
     })?;
     Ok(Epoch(epoch))
 }
+
+/// Sleep until the next epoch starts
+pub fn epoch_sleep(
+    test: &Test,
+    ledger_address: &str,
+    secs: u64,
+) -> Result<Epoch> {
+    let old_epoch = get_epoch(test, ledger_address)?;
+    let start = Instant::now();
+    let loop_timeout = Duration::new(secs, 0);
+    loop {
+        if Instant::now().duration_since(start) > loop_timeout {
+            panic!("Timed out waiting for the next epoch");
+        }
+        let epoch = get_epoch(test, ledger_address)?;
+        if epoch > old_epoch {
+            break Ok(epoch);
+        } else {
+            sleep(15);
```

Comment:
I think can probably be more often, the default epoch in e2e are quiet fast
```suggestion
            sleep(1);
```